docs/_docs/configuration.md

```diff
@@ -433,6 +433,7 @@ For example, suppose you set this conditional statement in your code:
 {% raw %}
 {% if jekyll.environment == "production" %}
    {% include disqus.html %}
+   {% include txtpen.html %}
```

Comment:
This is not the place to add an include. Also, the file to include in missing in this PR